docs/configuration.md

```diff
@@ -336,6 +323,8 @@ window.$docsify = {
 
 Website logo as it appears in the sidebar. You can resize it using CSS.
 
+!> Logo will only bee visible if `name` prop is also set. See [name configuration](configuration?id=name).
```

Comment:
Thanks for the PR.

But need to apply for merge to develop branch, not master. you can resolve the conflict or resubmit a new PR.

Reply:
Hello, It's already merging o develop branch

Reply:
<img width="832" alt="image" src="https://github.com/docsifyjs/docsify/assets/33931153/1895e77f-ae92-4c30-827b-543deba71abc">
It has some conflicts that need to be resolved.

Reply:
I've created another PR #2347